src/hooks/useTransactionList.ts

```diff
@@ -160,7 +160,7 @@ export function useTransactionList(wallet: EdgeCurrencyWallet, tokenId: EdgeToke
       cleanupChanged()
       cleanupStream()
     }
-  }, [searchString, spamThreshold, tokenId, wallet])
+  }, [searchString, tokenId, wallet])
```

Comment:
I think we still need `spamThreshold.current` in here. That way we re-run the effect if the spam threshold ref changes.